esx_service/utils/kvESX.py

```diff
@@ -298,7 +310,15 @@ def save(volpath, kv_dict):
     vol_name = vmdk_utils.get_volname_from_vmdk_path(volpath)
     while True:
         try:
+            if os.stat(meta_file).st_size > 0:
```

Comment:
I think this block of logging is redundant.



Agreed that this logs are temp and would be removed once we hit the root cause of the issue about the volume metadata  being unavailable(either corrupted or empty); do you mind wrapping this lines into a function and call it twice. The only reason being this change is being merged into master.